amr-wind/physics/ConvectingTaylorVortex.cpp

```diff
@@ -161,8 +162,10 @@ amrex::Real ConvectingTaylorVortex::compute_error(
 
 void ConvectingTaylorVortex::output_error()
 {
-    const amrex::Real u_err = compute_error(0, m_velocity, u_exact);
-    const amrex::Real v_err = compute_error(1, m_velocity, v_exact);
+    UExact u_exact;
+    VExact v_exact;
```

Comment:
you don't need these. You can remove the first and third argument from `compute_error` and just do 

```C++
compute_error<UExact>(m_velocity);
```

Reply:
let me try that!

Reply:
Ok I fixed that. Thanks for seeing that